PowerEditor/src/NppDarkMode.cpp

```diff
@@ -967,65 +1303,65 @@ namespace NppDarkMode
 
 		switch (uMsg)
 		{
-			case WM_UPDATEUISTATE:
-				if (HIWORD(wParam) & (UISF_HIDEACCEL | UISF_HIDEFOCUS))
-				{
-					InvalidateRect(hWnd, nullptr, FALSE);
-				}
-				break;
-			case WM_NCDESTROY:
-				RemoveWindowSubclass(hWnd, ButtonSubclass, g_buttonSubclassID);
-				delete pButtonData;
+		case WM_UPDATEUISTATE:
+			if (HIWORD(wParam) & (UISF_HIDEACCEL | UISF_HIDEFOCUS))
+			{
+				InvalidateRect(hWnd, nullptr, FALSE);
+			}
+			break;
+		case WM_NCDESTROY:
+			RemoveWindowSubclass(hWnd, ButtonSubclass, g_buttonSubclassID);
+			delete pButtonData;
```

Comment:
The 9 above added lines are identical as the 9 removed lines, please restore them.